Clarified third party reporting (#10244)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/user-submission.md

You can turn off the Microsoft integrated reporting experience to use third-party message reporting tools to send reported messages to the user submissions mailbox.

When **Microsoft Outlook Report Message button** is **Off** ![Toggle off.](../../media/scc-toggle-off.png) the following settings are available on the **User reported settings** page:

- **Use Microsoft's integrated Outlook reporting experience** section:

>   **Use this custom mailbox to receive user reported messages**: Select this option and enter the email address of an existing Exchange Online mailbox to read end user reports made using the third-party message reporting tools. _This reports won't be submitted to Microsoft but will be shown on user reported page under Actions & Submissions with result - Not Submitted to Microsoft. The result means that the end user reports made from third party reporting tools were not made to Microsoft and thus there is no result available._

Reviewer: ```suggestion
  **Use this custom mailbox to receive user reported messages**: Select this option and enter the email address of an existing Exchange Online mailbox to read end-user reports made using the third-party message reporting tools. _These reports won't be submitted to Microsoft but will be shown on the user reported page under Actions & Submissions with the result - Not Submitted to Microsoft. The result means that the end-user reports made from third-party reporting tools were not made to Microsoft and thus there is no result available._
```

> The only requirement while Microsoft is reading from that mailbox is that the original messages are included as uncompressed .EML or .MSG attachments in messages that are sent to user submissions mailbox. In other words, don't just forward the original messages to the user submissions mailbox.

Reviewer: ```suggestion
The only requirement while Microsoft is reading from that mailbox is that the original messages are included as uncompressed .EML or .MSG attachments in messages that are sent to the user submissions mailbox, so you should not just forward the original messages to the user submissions mailbox.
```
